### DISCUSSION TITLE
能加一个推荐出装么（笑着哭）

### DISCUSSION THREAD

**Discussion by hato233** [cangzhang/ChampR#246] [General]
如题，没这东西我都不会玩游戏了

**Comment by cangzhang** [comment]
![image](https://user-images.githubusercontent.com/1357073/206709272-74b5c498-5aff-47f8-80cb-cac3e82e9cf8.png)

这个功能不是一直有嘛

**Reply by hato233** [reply]
👌
这个我每次都点了;但是不知道是不是和wegame的出装推荐冲突了;每次的出装都很奇怪;下次去掉wegame的出装推荐我再试试🤣
